leshan-server-cf/src/main/java/org/eclipse/leshan/server/californium/impl/CaliforniumLwM2mRequestSender.java

```diff
@@ -61,16 +64,16 @@ public CaliforniumLwM2mRequestSender(final Set<Endpoint> endpoints, final Observ
     }
 
     @Override
-    public <T extends LwM2mResponse> T send(final Client destination, final DownlinkRequest<T> request, Long timeout)
-            throws InterruptedException {
+    public <T extends LwM2mResponse> T send(final Client registrationInfo, final DownlinkRequest<T> request,
```

Comment:
I thought of preparing here for the name changes we discussed. But here, yes, it still represents a destination. I can revert it.